packages/dart/lib/src/network/parse_live_query.dart

```diff
@@ -31,7 +31,7 @@ class Subscription<T extends ParseObject> {
   }
 }
 
-enum LiveQueryClientEvent { CONNECTED, DISCONNECTED, USER_DISCONNECTED }
+enum LiveQueryClientEvent { connected, disconnected, userDisconnected }
```

Comment:
Are these keys exposed to the developer? If so, could this be a breaking change?

Reply:
These internal functions are related to LiveQuery and are not used externally.
This change is based on the [variable naming](https://dart-lang.github.io/linter/lints/constant_identifier_names.html) rules in the Dart language